shell/platform/android/platform_view_android.cc

```diff
@@ -405,7 +417,8 @@ void PlatformViewAndroid::SetSemanticsEnabled(jboolean enabled) {
 
 void PlatformViewAndroid::ReleaseSurface() {
   NotifyDestroyed();
-  android_surface_->TeardownOnScreenContext();
+  if (android_surface_)
+    android_surface_->TeardownOnScreenContext();
```

Comment:
Should the PlatformView revert to the NullRasterizer here?

Reply:
Done